support-frontend/app/actions/CustomActionBuilders.scala

```diff
@@ -39,11 +39,11 @@ class CustomActionBuilders(
   private val idSkipValidationReturn: (String, String) = "skipValidationReturn" -> "true"
 
   private def idWebAppRegisterUrl(path: String, clientId: String, idWebAppRegisterPath: String): String =
-    idWebAppUrl.value / idWebAppRegisterPath ?
+    (idWebAppUrl.value / idWebAppRegisterPath ?
       ("returnUrl" -> s"$supportUrl$path") &
       idSkipConfirmation &
       idSkipValidationReturn &
-      "clientId" -> clientId
+      "clientId" -> clientId).toString
```

Comment:
toString makes me nervous (due to no type checking as it works on anything)
I assume that's the only way though rather than a toUrl or something

Reply:
I know what you mean, but that appears to be all we've got:
https://github.com/lemonlabsuk/scala-uri/blob/6b5bb3357b7d60e78514ae5664c9d4f1b46210a4/shared/src/main/scala/io/lemonlabs/uri/Uri.scala#L86

Reply:
no problem not much we can do then!